scripts/core/ui/components/ListPage/generic-list-page-item-view-edit.tsx

```diff
@@ -159,7 +166,7 @@ export class GenericListPageItemViewEdit<T> extends React.Component<IProps<T>, I
                 this.setState({
                     issues: {},
                 }, () => {
-                    this.props.onEditModeChange(false);
+                    this.props.onEditModeChange(true);
```

Comment:
are you sure this change is needed? It seems to me it was correct - after you're done editing - you should go into preview mode - thus editing mode should be set to `false`